atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/SweepStrategyManagers.java

```diff
@@ -45,9 +56,8 @@ public static SweepStrategyManager createDefault(KeyValueService kvs) {
                             .expireAfterAccess(1, TimeUnit.DAYS)
```

Comment:
do we want a max size on this cache to limit memory footprint?

Reply:
I don't think so, considering the default is to pre-warm the cache with all tables. The intended use case is that there won't be very many tables. If we had a max size here it seems like you could get very bad sweep perf once you have more than that amount of tables.